app/models/friendship.rb

```diff
@@ -0,0 +1,67 @@
+class Friendship < ActiveRecord::Base
+  extend TokenContext
+  include Tokenizer
+	enum friendship_status: [ :denied, :pending, :accepted]
+  attr_accessor :friendship_request_token
+	cattr_accessor :daily_request_limit
+  @@daily_request_limit = 12
+  belongs_to :user, :foreign_key => "user_id", :class_name => "User"
```

Comment:
Use the hash syntax here, not the rocket.  `foreign_key: 'user_id', etc.
Also, you should use single quotes when possible.  In ruby double quotes are more expensive because the interpreter has to check if any string interpolation is occurring. 
Lastly, your indent is off for the relation below this.